modern/src/common/util/preferences.js

```diff
@@ -39,3 +39,10 @@ export const useAttributePreference = (key, defaultValue) => useSelector((state)
   }
   return defaultValue;
 });
+
+export const useServerAttributePreference = (key, defaultValue) => useSelector((state) => {
```

Comment:
It doesn't make sense to add this function. You can simply write:
```js
state.session.server.attributes.key || defaultValue
```

Reply:
Thank you, did not know that.